lib/scrabble.rb

```diff
@@ -1,18 +1,35 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize
+    @word = nil
+  end
 
   def score(word)
-    1
+    @word = word.upcase!
+    word = sub_points(@word)
+    word = add_points(word)
+  end
+
+  def sub_points(arg)
+    point_values =
+      {
+        "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+        "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+        "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+        "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+        "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+        "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+        "Y"=>4, "Z"=>10
+      }
+
+     subbed_points = arg.split(//).map {|letter|
```

Comment:
Replacing `arg` with a more descriptive variable should help the readability of this method.